src/main/java/com/softserve/academy/spaced/repetition/controller/DeckController.java

```diff
@@ -120,7 +120,8 @@
     @Auditable(action = AuditingAction.CREATE_DECK_IN_COURSE)
     @PostMapping(value = "/api/category/{category_id}/courses/{course_id}/decks")
     @PreAuthorize(value = "@accessToUrlService.hasAccessToCourse(#category_id, #course_id)")
-    public ResponseEntity<DeckPublicDTO> addDeckToCourse(@RequestBody Deck deck, @PathVariable Long category_id, @PathVariable Long course_id) {
+    public ResponseEntity<DeckPublicDTO> addDeckToCourse(@RequestBody Deck deck,
+                                                         @PathVariable Long category_id, @PathVariable Long course_id) {
```

Comment:
Rename the variables in the camelCase, please.

Reply:
It is not my method

Reply:
Well, okay.